modules/chapter1/pages/index.adoc

```diff
@@ -1,19 +1,17 @@
 = Model Serving in RHODS
 
-In this chapter, we will describe the concepts and terminology around model serving on RHODS.
+== Introduction
+
+Generally speaking, an organization decides to train a model, because the organization wishes to empower the applications portfolio with inferences made by such model. In this way, the given known data points of a given event, can produce an inference, guess, or prediction of a topic of interest.
+
+In this course, you will learn how to leverage Red Hat OpenShift Data Science to serve machine learning models.
+
+How do we deliver a model to an inference engine, or server, so that, when the server receives a request from any of the applications in the organization's portfolio, the inference engine can reply with a prediction made based in the model that we have previously worked hard to train.
```

Comment:
This last paragraph, is it written as a question?

```suggestion
How do we deliver a model to an inference engine, or server, so that, when the server receives a request from any of the applications in the organization's portfolio, the inference engine can reply with a prediction made based in the model that we have previously worked hard to train?
```